### PR TITLE
Announcement for the aws tracks

### PR DESCRIPTION
**Pull request by aehyvari**

### CODE REVIEW

File: _news_2021/2021-02-05-parallel-and-cloud-tracks.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+<https://github.com/aws-samples/aws-batch-comp-infrastructure-sample>.
+
+
+### Parallel track
+
+The solvers participating in this track will be executed with a wall-clock time
+limit of 20 minutes, thus similar to the Single Query Track.  Each solver will
+be run on a single AWS machine of the type m4.16xlarge, which has 64 virtual
+cores and 256GB of memory. More details about m4.16xlarge nodes can be found
+[here](https://aws.amazon.com/about-aws/whats-new/2016/09/introducing-new-m4-instance-size-m4-16xlarge-and-new-region-availability-of-m4-instances/).
```
Comment (HanielB) [inline _news_2021/2021-02-05-parallel-and-cloud-tracks.md:59]: Add an entry like that for details for the m4.4xlarge as well, above?

Comment (aehyvari) [reply _news_2021/2021-02-05-parallel-and-cloud-tracks.md:59]: There was no link in the source document that I got from sat-competition for m4.4xlarge nodes.  I guess they're not that special to warrant a press release from aws, or at least I couldn't find one by quick googling.

Comment (jhoenicke) [reply _news_2021/2021-02-05-parallel-and-cloud-tracks.md:59]: What I found is this: https://aws.amazon.com/ec2/instance-types/ 

But you have to click yourself through (m4 is listed in the first table).

On the other hand the 16 core + 64 MB memory is the most important information and already included, so it's not really necessary to link it.

Comment (aehyvari) [reply _news_2021/2021-02-05-parallel-and-cloud-tracks.md:59]: Yes.  I'd just leave it as it is there now not to complicate things.  I'm guessing that the point of having the link to the press release is to highlight (in a subtle way) that the parallel track runs in the top-end nodes, which is a cool thing in itself.

**Review by jhoenicke — APPROVED** [review]
LGTM

**Review by HanielB — APPROVED** [review]

**Pull request merged**